jenkins/tenant/aws/common/run_policies.py

```diff
@@ -1,10 +1,34 @@
-
 import os
 from ast import literal_eval
 
-policies_in_action = []
-policies_not_in_action = ['ec2_stop', 'ec2_idle', 'zombie_cluster_resource', 'ebs_unattached', 'ip_unattached',
-                          'zombie_snapshots', 'unused_nat_gateway', 's3_inactive', 'empty_roles']
+from cloud_governance.common.tool.tool import get_custodian_policies
+
+exclude_policies = ['cost_explorer', 'optimize_resources_report', 'monthly_report', 'cost_over_usage',
+                    'skipped_resources', 'cost_explorer_payer_billings', 'cost_billing_reports', 'spot_savings_analysis']
+available_policies = get_custodian_policies(exclude_policies=exclude_policies)
```

Comment:
get_custodian_policies ?

Reply:
Sry, it is get policies. Get all policies with excluding the cost policies.